data/syncer/baseAccountsSyncer.go

```diff
@@ -128,7 +128,8 @@ func (b *baseAccountsSyncer) printStatistics(ssh data.SyncStatisticsHandler, ctx
 				"num received", ssh.NumReceived(),
 				"num missing", ssh.NumMissing(),
 				"num tries", fmt.Sprintf("%d/%d", atomic.LoadInt32(&b.numTriesSynced), atomic.LoadInt32(&b.numMaxTries)),
-				"intercepted trie nodes cache", fmt.Sprintf("len: %d, size: %s", b.cacher.Len(), core.ConvertBytes(b.cacher.SizeInBytesContained())))
+				"intercepted trie nodes cache", fmt.Sprintf("len: %d, size: %s", b.cacher.Len(), core.ConvertBytes(b.cacher.SizeInBytesContained())),
+				"num of intercepted trie nodes", b.cacher.Len())
```

Comment:
you use b.cacher.Len() in both prints: when printing the size and when printing the number of elements. Is this correct?

Reply:
Refactored.